MC-39773: [Documentation] Editing Price Rule itself affects current scheduled update (#1017)
magento/merchdocs: src/cms/content-staging.md

   If all campaigns have end dates, the baseline content is restored whenever all active campaigns end.

{:.bs-callout-info}

> While content staging update is active for any supported entity, editing the entity itself means editing the current active staging update, it does not affect the baseline content, which will be restored when the staging update ends.  

Reviewer: ```suggestion
While a staging update is active for an entity, editing the entity is editing the current active staging update. It does not affect the baseline content, which is restored when the staging update ends.  
```
